packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -793,6 +795,50 @@ export class DbIndexEntries {
   ) {}
 }
 
+export type DbDocumentOverlayKey = [string, string, string];
```

Comment:
Please add comment explaining each string.

Easiest is: `[/* foo=* / string, /* foo=* / string, /* foo=* / string]`